### PR TITLE
BZ2054750: added annotation

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.9 and above vesions
- [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=2054750)
- [Preview](https://deploy-preview-42412--osdocs.netlify.app/openshift-enterprise/latest/networking/hardware_networks/installing-sriov-operator.html#install-operator-web-console_installing-sriov-operator)
- @zhaozhanqi  ptal

### CODE REVIEW

File: modules/nw-sriov-installing-operator.adoc [hunk 1]
```diff
@@ -148,5 +148,10 @@ If the operator does not appear as installed, to troubleshoot further:
...
 +
 * Inspect the *Operator Subscriptions* and *Install Plans* tabs for any failure or errors under *Status*.
-* Navigate to the *Workloads* -> *Pods* page and check the logs for pods in the
-`openshift-sriov-network-operator` project.
+* Navigate to the *Workloads* -> *Pods* page and check the logs for pods in the `openshift-sriov-network-operator` project.
+* Check the namespace of the YAML file and if the annotation is missing, you can add the annotation `workload.openshift.io/allowed=management` to the Operator namespace with the following command:
++
+[source,terminal]
+----
+$ oc annotate ns/openshift-sriov-network-operator workload.openshift.io/allowed=management
```
Comment (zhaozhanqi) [inline modules/nw-sriov-installing-operator.adoc:156]: I think this only needed for SNO cluster.  if so, we need reference with `SNO` cc  yliu1@redhat.com

File: modules/nw-sriov-installing-operator.adoc [hunk 2]
```diff
@@ -148,5 +148,10 @@ If the operator does not appear as installed, to troubleshoot further:
 
 +
 * Inspect the *Operator Subscriptions* and *Install Plans* tabs for any failure or errors under *Status*.
-* Navigate to the *Workloads* -> *Pods* page and check the logs for pods in the
-`openshift-sriov-network-operator` project.
+* Navigate to the *Workloads* -> *Pods* page and check the logs for pods in the `openshift-sriov-network-operator` project.
+* Check the namespace of the YAML file and if the annotation is missing, you can add the annotation `workload.openshift.io/allowed=management` to the Operator namespace with the following command:
```
Comment (mjpytlak) [inline modules/nw-sriov-installing-operator.adoc:152]:
```suggestion
* Check the namespace of the YAML file. If the annotation is missing, you can add the annotation `workload.openshift.io/allowed=management` to the Operator namespace with the following command:
```

**Comment by Amrita42** [comment]
@zhaozhanqi  ptal at the new line of content based on your feedback.

**Pull request merged**